### DISCUSSION TITLE
Can CP unload a combine driven by human player ?

### DISCUSSION THREAD

**Discussion by Lactic68** [Courseplay/Courseplay_FS22#2507] [General]
If we would like to drive the combine, we have set up a CP to watch the field for unload the combine but nothing happen when the combine is full.
Does this work only if the combine is driven by CP too ?

**Comment by Tensuko** [comment]
No, it can't due to the system of following the combine course.
There is no way the unloader can follow a player since a player doesn't have a predictable course.

**Reply by Lactic68** [reply]
Ok, thank you for your quick answer :)

**Comment by Xenon1029** [comment]
what if you load a combine course then drive it yourself?

**Reply by Tensuko** [reply]
Still not.
A palyer driven combine is not a CP driven combine, a loaded course won't change anything there.
If you want to drive the combine by yourself and have it unloaded automatically, use AD.

**Comment by MaikoRK** [comment]
hello there.. damm so sad to hear thats this option is not possible, its the most amazing thing i miss from previous courseplay releases from fs19 17.. etc
cuz i dont like to 100% automatize the field work, and harvesting is definitely something i love to do, just wish i could use courseplay to call an auger to unload it while harvesting S2 S2
so why is it not possible?? previous versions of courseplay it could be done.. inst there a way to add this feature again??

**Reply by Tensuko** [reply]
Because the unloader is driving on a copy of the combine course.

**Reply by MaikoRK** [reply]
sure i know that.. i just want to see if it possible for you to use similar mechanics from fs17 or before that allowed us to harvest by yourself and use courseplay for augers and trailers??

**Reply by Tensuko** [reply]
No. It will not come back!
I answerd it above already...
